packages/server-wallet/src/chain-service/__chain-test__/chain-service.test.ts

```diff
@@ -203,45 +204,11 @@ describe('fundChannel', () => {
     const channelId = randomChannelId();
 
     await waitForChannelFunding(0, 5, channelId, erc20AssetHolderAddress);
-    const contract: Contract = new Contract(
-      erc20AssetHolderAddress,
-      ContractArtifacts.Erc20AssetHolderArtifact.abi,
-      provider
-    );
-    expect(await contract.holdings(channelId)).toEqual(BigNumber.from(5));
+    expect(await erc20Holder.holdings(channelId)).toEqual(BigNumber.from(5));
   });
 });
 
 describe('registerChannel', () => {
-  it('dispatches a channel finalized event if the channel has been finalized BEFORE registering', async () => {
```

Comment:
I'm not seeing why this test case is redundant?

Reply:
While the goal of this test was to test what happens to a channel that is finalized before registering I ran into issues with `ganache` and the timestamp.

This resulted in a commented out expectation which means this test will behave exactly like the other test.
```
// TODO: Currently due to ganache mining blocks outside of our control we can't assert on this	
    // expect(channelFinalizedHandler).not.toHaveBeenCalled();
```

Reply:
got it, thanks